ci/steps/attach_logs.py

```diff
@@ -187,14 +253,25 @@ def upload_logs(
         only_failed=False,
     )
     if ok:
-        _attach_and_upload_logs(
+        ok = _attach_and_upload_logs(
             architecture=architecture,
+            build_dict_json=build_dict_json,
             build_targets=build_targets,
             cicd_cfg_name=cicd_cfg_name,
             committish=committish,
+            flavour_set=flavourset,
             gardenlinux_epoch=gardenlinux_epoch,
-            modifiers=modifiers,
-            platform=platform,
+            is_package_build='gl-packages' in pipeline_run_name,
+            manifest_set_key=manifest_set_key,
+            platform_set=platform_set,
+            promote_target=promote_target,
             repo_dir=repo_dir,
             version=version,
         )
+
+    print(f'Upload logs finished with result {ok}')
+    if not ok:
```

Comment:
_could_ be shortened to
```python3
    if not _attach_and_upload_logs(
        architecture=architecture,
        build_dict_json=build_dict_json,
        build_targets=build_targets,
        cicd_cfg_name=cicd_cfg_name,
        committish=committish,
        flavour_set=flavourset,
        gardenlinux_epoch=gardenlinux_epoch,
        is_package_build='gl-packages' in pipeline_run_name,
        manifest_set_key=manifest_set_key,
        platform_set=platform_set,
        promote_target=promote_target,
        repo_dir=repo_dir,
        version=version,
    ):
```
Up to you.

Reply:
Agreed, but ok variable is used before. In this case we would need to double logic in case of failure.